app/src/components/modal/common_styled/index.tsx

```diff
@@ -91,6 +91,9 @@ export const BalanceItem = styled.div<{ hover?: boolean }>`
   }
 
   &:hover {
+    .hover {
```

Comment:
here we can implement a similar but different solution. We can import `TextWrapper` from the theme and target it `${TextWrapper` inside the :hover call. Though also the token balances will get the hover effect, which in this case looks also good. But the first solution, implemented here, is more flexible to use now andin the future cause we simply have to add the className `hover` to activate the hover when we wish within a `BalanceItem`

Reply:
So the reason that It doesn't work currently is because its selecting balanceItemSide and BalanceItemTitle for hover and you have replaced them with TYPE.sometihing....I would suggest to leave those classes and handle them like airdrop issue if possible. Or try to target those classes also you have to be careful when changing and deleting styled components like this because In some cases we select them based on styled class like here.
![Screenshot 2021-08-25 at 18 23 37](https://user-images.githubusercontent.com/33226956/130828270-dcb22c91-e430-4f9f-93bf-314588afa323.png)
Also why are you using theme.colors.textColorDark when that is theme.text1 like the item that is below in this picture....